src/app/tests/suites/certification/Test_TC_SC_4_2.yaml

```diff
@@ -85,7 +85,6 @@ tests:
 
     - label: "Check Hostname"
       # On macOS the hostname is the device name and because of it this test is disabled for now.
```

Comment:
Why is this comment still here?

Reply:
@bzbarsky-apple forget to remove the comment i will update it